js/src/Lines.ts

```diff
@@ -632,15 +632,29 @@ export class Lines extends Mark {
   }
 
   get_interpolation():
+    | d3Shape.CurveFactory
+    | d3Shape.CurveFactory
+    | d3Shape.CurveFactory
+    | d3Shape.CurveFactory
+    | d3Shape.CurveFactory
+    | d3Shape.CurveFactory
+    | d3Shape.CurveFactory
     | d3Shape.CurveFactory
     | d3Shape.CurveFactory
     | d3Shape.CurveFactory
     | d3Shape.CurveFactory {
     const curveTypes = {
```

Comment:
This type definition looks odd. Can it just be this?
```suggestion
  get_interpolation(): d3Shape.CurveFactory {
    const curveTypes = {
```

Reply:
Sure, I kept the old syntax, but this looks much better